### PR TITLE
Add duplicates count to possible conditions

### PR DESCRIPTION
**Pull request by norganna**
Allows one to do similar to the following:

```
>> copy = original
>> copy.dedup()
Dedupped 2080, kept 37
>> copy.search("dups > 5")
```

### CODE REVIEW

File: goroutine.go [hunk 1]
```diff
@@ -407,11 +407,12 @@ func (gd *GoroutineDump) withCondition(cond string, callback func(int, *Goroutin
...
-			"duration": g.duration,
-			"lines":    g.lines,
-			"state":    g.metas[MetaState],
-			"trace":    g.trace,
+			"id":         g.id,
+			"duration":   g.duration,
+			"lines":      g.lines,
+			"state":      g.metas[MetaState],
+			"trace":      g.trace,
+			"duplicates": len(g.duplicates),
```
Comment (linuxerwang) [inline goroutine.go]: Better move up above the "duration" line. What do you think make it shorter, like "dup"?

Comment (norganna) [reply goroutine.go]: I have moved it up.

**Comment by norganna** [comment]
> Better move up above the "duration" line. What do you think make it shorter, like "dup"?

Willing to make it "dup" (or perhaps "dups" for consistency with "lines" etc) if you prefer that.

**Comment by linuxerwang** [comment]
> > Better move up above the "duration" line. What do you think make it shorter, like "dup"?
> 
> Willing to make it "dup" (or perhaps "dups" for consistency with "lines" etc) if you prefer that.

"dups" sounds better.

**Comment by norganna** [comment]
> "dups" sounds better.

Done.

**Comment by linuxerwang** [comment]
> > "dups" sounds better.
> 
> Done.

Thanks!

**Pull request merged**